adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -56,6 +56,10 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(AbstractVertxBasedHttpProtocolAdapter.class);
 
+    private static final String HEADER_QOS_LEVEL = "QoS-Level";
```

Comment:
I think we should move this to `org.eclipse.hono.util.Constants` so that client code can easily refer to it as well.

Don't forget to update your test cases accordingly ...

Reply:
sure, thanks! looking at the copyright section of `notice.md`, i did not see individual names added there. how does this work with students?

Reply:
just add your name if you are not doing the work in the context of an organization